notification-components/js/notification.js

```diff
@@ -0,0 +1,98 @@
+export async function getData() {
+    const disabledNotification = localStorage.getItem('disabledNotification') || 'false';
+    if (disabledNotification === 'false') {
+        const currentTip = localStorage.getItem('currentTip') || 1;
+        const response = await fetch('data.json');
+        const tips = await response.json();
+        const {title, phrase} = tips.find(tip => tip.id === parseInt(currentTip));
+        document.body.innerHTML = `<div class="notificationPage">
+                                        <div class="notification">
+                                            <div class="notificationCard">
+                                                <div class="notificationText">
+                                                    <h3 id="title">${title}</h3>
+                                                    <p id="phrase">${phrase}</p>
+                                                </div>
+                                                <div class="notificationCloseButton">
+                                                    <i id="closeButton" class="fas fa-times" tabindex="0"></i>
+                                                </div>
+                                            </div>
+                                            <div>
+                                                <div class="container">
+                                                    <ul class="pagination">
+                                                        <li><a id="tipPrevious" href="#">Previous</a></li>
```

Comment:
butons should be `<button>` elements!
link `<a>` reloads the page, page state is lost (you can see that focus is lost for example)
```suggestion
                                                        <li><button id="tipPrevious">Previous</button></li>
```